.github/renovate.json

```diff
@@ -2,8 +2,11 @@
   "$schema": "https://docs.renovatebot.com/renovate-schema.json",
   "dependencyDashboard": true,
   "suppressNotifications": ["prEditedNotification"],
-  "extends": ["config:base"],
+  "extends": ["config:recommended"],
   "labels": ["bot: dependencies"],
+  "semanticCommits": "disabled",
```

Comment:
This means PR titles won't start with the `(chore)` or `(fix)` prefix. I find them really annoying, and we don't use them generally for PRs made by humans